apps/charterafrica/src/components/Tool/DescriptionAndShare.js

```diff
@@ -21,10 +22,12 @@ const DescriptionAndShare = React.forwardRef(
         <RichTypography
           textAlign="left"
           color="neutral.dark"
-          sx={{ mt: 3.75 }}
+          sx={{ mt: 3.75, textAlign: { xs: "center", sm: "left" } }}
           variant="p1"
         >
-          {lastActive}
+          {formatDateTime(lastActive, {
+            includeTime: true,
+          })}
```

Comment:
Shouldn't this be formatted in the backend i.e. somewhere in `lib/` including `locale` information?